docker/base/Dockerfile

```diff
@@ -32,27 +30,62 @@ RUN echo 'Etc/UTC' > /etc/timezone && \
 
 RUN apt-get update \
  && apt-get install -y \
-        build-essential \
+        cmake \
+        cppcheck \
         curl \
+        gdb \
```

Comment:
Nit : Do we need gdb, and libbluetooth for the docker ?

Reply:
Ahh, my mistake. I'll open up a follow-up PR.